AbletonMCP_Remote_Script/handlers/session.py

```
@@ -0,0 +1,1124 @@
+        return {"current_song_time": song.current_song_time}
+    except Exception as e:
+        if ctrl:
+            ctrl.log_message("Error setting playback position: " + str(e))
+        raise
+
+
+def set_arrangement_overdub(song, enabled, ctrl=None):
+    """Enable or disable arrangement overdub mode."""
+    try:
+        song.arrangement_overdub = enabled
+        return {"arrangement_overdub": song.arrangement_overdub}
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)